packages/ERTP/test/unitTests/mathHelpers/test-copyBagMathHelpers.js

```diff
@@ -170,6 +170,29 @@ test('copyBag with strings add', t => {
   );
 });
 
+test('copyBag with strings min/max', t => {
```

Comment:
I wonder about the value of min/max tests in each mathHelpers; they're redundant with `isGTE` tests (or should be).

A test to verify the availability of the `min/max` API should suffice, I think.

Reply:
One thing that would be quite nice is to take the property-based testing for bag amounts #4366 and generalize to all amounts.

Reply:
> I wonder about the value of min/max tests in each mathHelpers; they're redundant with `isGTE` tests (or should be).
> 
> A test to verify the availability of the `min/max` API should suffice, I think.

I don't understand either of these comments. The way the PR looks now, with `min` and `max` defined once, simply, in terms of `isGTE` seems exactly right. We can leave the helpers out of this.

Reply:
What I mean is: it's worth having a test that verifies that `AmountMath.min` exists and works as designed in a simple case or two; likewise `AmountMath.max`. That's what I meant by a test to verify the availability of the min/max API.

Lots of tricky min/max tests don't seem worthwhile; the definition of min/max in terms of isGTE is sufficiently simple that any effort one might put into testing min/max should be put into testing isGTE instead.